rtloader/three/three.cpp

```diff
@@ -395,6 +395,41 @@ bool Three::getCheck(RtLoaderPyObject *py_class, const char *init_config_str, co
     return true;
 }
 
+bool Three::isCheckInitDeprecated(RtLoaderPyObject *py_class)
+{
+    PyObject *klass = reinterpret_cast<PyObject *>(py_class);
+    PyObject *init, *func_code, *co_varnames, *elt;
+    bool result = false;
+
+    // AgentCheck.__init__.__code__.co_varnames
```

Comment:
This will not work as `co_varnames` also includes local variables.

ie:
```python
def test():
    agentConfig=1

print test.func_code.co_varnames
('agentConfig',)
```

Reply:
You're right, I need to use `co_argcount` as well. I added a test to make sure it worked.